adminSiteClient/EditorTextTab.tsx

```diff
@@ -55,6 +55,24 @@ export class EditorTextTab extends React.Component<{ editor: ChartEditor }> {
         grapher.relatedQuestions.splice(idx, 1)
     }
 
+    @action.bound onToggleTitleAnnotationEntity(value: boolean) {
+        const { grapher } = this.props.editor
+        grapher.hideTitleAnnotation = grapher.hideTitleAnnotation || {}
```

Comment:
we can use modern JS features here:
```suggestion
        grapher.hideTitleAnnotation ??= {}
```

or, alternatively, if you like it more:
```ts
        grapher.hideTitleAnnotation = { ...grapher.hideTitleAnnotation, entity: value || undefined }
```